karma.conf.js

```diff
@@ -1,6 +1,6 @@
 // @AngularClass
 
-module.exports = function(config) {
+module.exports = function (config) {
```

Comment:
Likely strange behaviour from my auto-document formatter. I will go strip it out and up the a2d3 version.